src/Adapters/Auth/index.js

```diff
@@ -41,36 +51,40 @@ const providers = {
   vkontakte,
   qq,
   wechat,
-  weibo
-}
-
-function authDataValidator(adapter, appIds, options) {
-  return function(authData) {
-    return adapter.validateAuthData(authData, options).then(() => {
-      if (appIds) {
+  weibo,
+  xiaomi
+};
+
+function authDataValidator(adapter, appIds, options)
+{
+  return function (authData)
+  {
+    return adapter.validateAuthData(authData, options).then(() =>
+    {
+      if (appIds)
+      {
         return adapter.validateAppId(appIds, authData, options);
       }
       return Promise.resolve();
     });
-  }
+  };
 }
 
 function loadAuthAdapter(provider, authOptions) {
   const defaultAdapter = providers[provider];
   const adapter = Object.assign({}, defaultAdapter);
   const providerOptions = authOptions[provider];
 
-  if (!defaultAdapter && !providerOptions) {
+  if (!defaultAdapter && !providerOptions)
```

Comment:
Its recommended to always use brackets for if / else statements. Without them they can lead to bugs. You might want to go over your code again for consistency